lib/boombox/pipeline.ex

```diff
@@ -117,6 +117,31 @@ defmodule Boombox.Pipeline do
     proceed(ctx, state)
   end
 
+  @impl true
+  def handle_child_playing(:udp_source, ctx, state) do
+    Boombox.RTP.udp_ready(state.input, ctx, state)
+    {[], state}
+  end
+
+  def handle_child_playing(:rtsp_source, ctx, state) do
+    Boombox.RTP.udp_ready(state.input, ctx, state)
+    {[], state}
+  end
+
+  def handle_child_playing(_child, _ctx, state) do
+    {[], state}
+  end
+
+  @impl true
+  def handle_child_setup_completed(:udp_rtp_sink, ctx, state) do
```

Comment:
AFAIK `:udp_rtp_sink` is the UDP client, so I don't think we need to wait for it. We just want to make sure that when `Boombox.async()` returns, "external" clients can interact with Boombox.